src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -734,8 +732,6 @@ public boolean isLoggable(LogRecord record) {
          *
          * @noinspection CanBeFinal
          */
-        @Option(names = {"-W", "--tree-walker-threads-number"}, description = "(experimental) The "
-                + "number of TreeWalker threads (must be greater than zero)")
         private int treeWalkerThreadsNumber = DEFAULT_THREAD_COUNT;
```

Comment:
please put one sentence in javadoc of this field that it is skipped from CLI intentionally.

Reply:
@romani Done